scala/scala.bzl

```diff
@@ -730,8 +750,8 @@ def _scala_binary_common(ctx, cjars, rjars, transitive_compile_time_jars, jars2l
 
   # _java_toolchain added manually since _java doesn't currently setup runfiles
```

Comment:
is this comment incorrect anymore?

Reply:
Good catch, this comment is incorrect (as is one above it as well).  I'm actually checking into whether I need _java_toolchain here at all...will update when I have an answer (should be by tomorrow).

Reply:
Just to make things clear, I've updated the two comments I intend to update once I've resolved the _toolchain question.